scripts/reindex.py

```diff
@@ -58,6 +59,9 @@
                     ],
                     choices=config.catalogs,
                     help='The names of the catalogs to reindex.')
+parser.add_argument('--source',
+                    nargs='?',
```

Comment:
Seems like 
```suggestion
                    nargs='+',
```

is low hanging fruit. If it can be done easily please do so.

Reply:
`'+'` seems to be incompatible with allowing an empty `--source` arg but `'*'` works.